docs/collections/documents/tutorials/c8ql-docs-queries-sdk.md

```diff
@@ -0,0 +1,247 @@
+
+# Authenticate and log in
+print("--- Connecting to GDN")
+
+# Authenticate with API key
+client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+# Create HTTPS session
+session = requests.session()
+session.headers.update({"content-type": 'application/json'})
+session.headers.update({"authorization": AUTH_TOKEN})
+url = f"{HTTP_URL}/_fabric/_system/_api/cursor"
+
+# Insert documents to the collection
+resp = session.post(url, json={
+    "query": "INSERT{'name' : 'Julie', 'company' : 'ABC', '_key' : 'Julie'}" \
+             "INTO testcollection"
+})
+resp_json = resp.json()
+if resp_json.get("error") == False:
+    print("Document inserted successfully.")
+
+# Read from the collection
+resp = session.post(url, json={
+    "query": "FOR doc IN testcollection RETURN doc"
+})
+resp_json = resp.json()
+if resp_json.get("error") == False:
+    print("Documents retrieved successfully. Data: ")
+    for doc in resp_json.get("result", []):
+        print(json.dumps(doc, indent=4))
+
+# Update documents in the collection
+resp = session.post(url, json={
+    "query": "FOR c IN testcollection UPDATE c WITH{'company':'XYZ'} IN testcollection"
+})
+resp_json = resp.json()
+if resp_json.get("error") == False:
+    print("Document updated successfully.")
+
+# Upsert documents in the collection
+resp = session.post(url, json={
+    "query": "UPSERT {name: 'John'} INSERT "
+             "{_key:'John', name: 'John', logins:1, updatedAt: DATE_NOW()}"
+             " UPDATE {'logins': OLD.logins + 1, updatedAt: DATE_NOW()} IN testcollection"
+})
+resp_json = resp.json()
+if resp_json.get("error") == False:
+    print("Document upserted successfully.")
+
+# Delete documents in the collection
+resp = session.post(url, json={
+    "query": "FOR c IN testcollection REMOVE c IN testcollection"
+})
+resp_json = resp.json()
+if resp_json.get("error") == False:
+    print("Documents deleted successfully.")
+
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript SDK">
+
+```js
+class APIRequest {
+    _headers = {
+      Accept: "application/json",
+      "Content-Type": "application/json"
+    };
+  
+    constructor (httpUrl, apiKey) {
+      this._url = httpUrl;
+      this._headers.authorization = `apikey ${apiKey}`; // apikey keyword is needed here
+    }
+  
+    _handleResponse (response, resolve, reject) {
+      if (response.ok) {
+        resolve(response.json());
+      } else {
+        reject(response);
+      }
+    }
+  
+    req (endpoint, { body, ...options } = {}) {
+      const self = this;
+      return new Promise(function (resolve, reject) {
+        fetch(self._url + endpoint, {
+          headers: self._headers,
+          body: body ? JSON.stringify(body) : undefined,
+          ...options
+        }).then((response) => self._handleResponse(response, resolve, reject));
+      });
+    }
+  }
+  
+  const apiKey = "XXXXX" // Use your API key here
+  const globalUrl = "api-play.paas.macrometa.io";
```

Comment:
Can we just use one URL, for example, a federation URL, for this code sample? global = http = federation for this sample.

Reply:
@diana-macrometa first of all you are not using the SDKs here at all.. the previous page that was deleted was only using APIs but the now this page says "SDK" but the examples are for direct API calls.. so I don't understand the requirement here.. you can remove term SDK id we want to show users c8ql queries with direct API calls... Please clarify this first..

Reply:
The goal is to make this old piece of documentation relevant and useful for the current GDN.